### ISSUE TITLE
Location name is not accessible in next hour forecast

### ISSUE THREAD

**Issue by geosaaga** [fmidev/weather-app#976]
Location name can't be selected in Accessibility Inspector. Make it accessible.

**Comment by geosaaga** [comment]
PR: https://github.com/fmidev/weather-app/pull/978
